src/datasource.ts

```diff
@@ -543,7 +543,8 @@ export class OpenSearchDatasource extends DataSourceWithBackend<OpenSearchQuery,
               metric.type === 'raw_document' ||
               (request.app === CoreApp.Explore && target.queryType === QueryType.Lucene)
           ) ||
-          (request.app === CoreApp.Explore && target.queryType === QueryType.PPL && target.format === 'logs')
+          (request.app === CoreApp.Explore && target.queryType === QueryType.PPL && target.format === 'logs') ||
+          (request.app === CoreApp.Explore && target.queryType === QueryType.Lucene)
```

Comment:
Can we limit this to just the trace spans as the trace list is not yet fully in the backend? thanks!